CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ NOTE: This release is only compatible with Realm Object Server 3.21.0 or later.
 
 ### Enhancements
 * Add an optional parameter to the `SubscriptionOptions`: `inclusions` which is an array of linkingObjects properties. This tells subscriptions to include objects linked through these relationships as well (links and lists are already included by default). ([#2296](https://github.com/realm/realm-js/pull/2296)
+* Support 64 bit for React Native Android. ([#2221](https://github.com/realm/realm-js/issues/2221)
```

Comment:
@nhachicha should this have been added to the top of the changelog instead of the release nodes for 2.27.0-rc.3?